scriptmodules/libretrocores/lr-pcsx-rearmed.sh

```diff
@@ -48,7 +48,7 @@ function configure_lr-pcsx-rearmed() {
     ensureSystemretroconfig "psx"
 
     # system-specific, PSX
-    iniConfig " = " "" "$configdir/psx/retroarch.cfg"
+    iniConfig " = " '"' "$configdir/psx/retroarch.cfg"
     iniSet "rewind_enable" "false"
```

Comment:
I don't believe this config will work as it shows up after the #include line in the system specific config. It is likely that we'll need to tweak or add a function to included system based configs above the #include line. Would also be helpful for mednafen type cores until upstream issues are fixed.